tests/ttnn/unit_tests/operations/test_moreh_linear.py

```diff
@@ -92,6 +92,10 @@ def moreh_linear(shapes, has_bias, has_output, compute_kernel_config, device):
 @pytest.mark.parametrize("has_bias", [False, True])
 @pytest.mark.parametrize("compute_kernel_options", compute_kernel_options, ids=compute_kernel_ids)
 def test_moreh_linear(shapes, has_bias, compute_kernel_options, device):
+    # TODO: Duong Remove skiptest until PR https://github.com/tenstorrent/tt-metal/pull/12794
```

Comment:
This PR is now merged

Reply:
I have update the matmul code to support fp32 dst acc mode for fuse bias and remove this skip test. Thank you.